Automattic/jetpack
# WordPress.com Toolbar: Add Site Monitoring menu item

Fixes #https://github.com/Automattic/dotcom-forge/issues/3391
Related to: https://github.com/Automattic/wpcomsh/pull/1464

## Proposed changes:
* Jetpack plugin can add a new `Site Monitoring` menu item to the Tools menu
* Jetpack plugin removes the `Site Logs` menu item

<img width="438" alt="Screenshot 2023-08-14 at 3 45 11 PM" src="https://github.com/Automattic/jetpack/assets/25575134/1062c22f-2650-4653-9101-40fb74e9b393">

At the moment, the `Site Monitoring` menu item is added through WPCOMSH:  https://github.com/Automattic/wpcomsh/pull/146 . We would like to add it through Jetpack to be consistent with other menu items and remove it from WPCOMSH. 

**Additional Notes**

* Once this PR is deployed and the `Site Monitoring` menu item is added with Jetpack, we will remove the code from WPCOMSH (at the moment, the code makes a conditional check to see if the menu item is added by Jetpack and if not, it adds it. If yes, it skips adding it). 

### Other information:



## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

**To test on Atomic sites**
* Navigate to your Atomic Site on WordPress.com
* Add the [Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/)
* Once the plugin is active, go to `Jetpack > Jetpack Beta > Jetpack` and enable the `add/site-monitoring-menu-item-for-atomic` branch.
* Navigate to **Tools** menu item and confirm that **Site Monitoring** appears under **Tools** once and **is not** added a second time

**To check tests**:

* Pull the changes from this branch locally
* Run `jetpack docker phpunit -- --filter=Test_Atomic_Admin_Menu` in your terminal to confirm that all the tests are passing
* The expected outcome from running tests in that file is:

<img width="342" alt="Screenshot 2023-08-18 at 9 56 30 AM" src="https://github.com/Automattic/jetpack/assets/25575134/341d7d2c-b82f-46ba-acc7-0ad899fe855c">
